src/main/kotlin/uk/gov/dluhc/printapi/service/SftpService.kt

```diff
@@ -51,4 +54,18 @@ class SftpService(
         sftpOutboundTemplate.rename("$fileDirectoryPath/$originalFileName", "$fileDirectoryPath/$newFileName")
         return newFileName
     }
+
+    /**
+     * Fetches the file from the path on the remote server
+     * @param filePathToProcess the path to the file on the remote server
+     * @return the contents of the remote file
+     */
+    fun fetchFile(filePathToProcess: String): String {
```

Comment:
1. To be in line with other method you could rename this to `fetchFileFromOutBoundDirectory`?
2. Ideally you could pass the `directory` and `fileName` to this method. Same for `removeMethod`

Reply:
I've updated the method names as you suggested 👍